examples/005_histogram_fit/histogram_fit.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+
+"""
+kafe2 example: Histogram Fit
+=======================
+
+kafe2 is not only capable of performing XY-Fits. This example shows how to handle a one dimensional dataset.
```

Comment:
First sentence sounds weird.
Second sentence makes it sound like HistFit is the way to handle all one-dimensional data in kafe2 (IndexedFit).
I suggest:

In addition to handling xy data via XYFit kafe2 can also bin one-dimensional data to create a histogram fit.

Reply:
Good point about `HistFit` not being the only way to handle 1d data. The second sentence could be made more clear by e.g. "One way to handle one-dimensional data with kafe2 is by fitting a **histogram**."